lib/src/main/kotlin/com/swmansion/starknet/data/types/CairoVersion.kt

```diff
@@ -0,0 +1,19 @@
+package com.swmansion.starknet.data.types
+
+import com.swmansion.starknet.extensions.toFelt
+
+enum class CairoVersion(val version: Felt) {
+    ZERO(Felt.ZERO), ONE(Felt.ONE);
```

Comment:
I am never sure if using only option 0 or 1 for the Cairo version is not confusing for users. Could we add a documentation stating that 0 corresponds to the legacy version and 1 corresponds to any contract compiled with Cairo >=1?

Reply:
Sure, sounds good to me.